res/shaders/rgbsignal.frag

```diff
@@ -5,6 +5,7 @@ uniform vec4 axesColor;
 uniform vec4 lowColor;
 uniform vec4 midColor;
 uniform vec4 highColor;
+uniform bool splitStereoSignal;
```

Comment:
@m0dB do you what happens if the shader program doesn't bind the uniform? Is there a consistent default value? Just wondering what to do in the deprecated renderer